setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+VERSION = '1.0'
+
+from io import open
+from os import path
+from setuptools import setup
+
+here = path.abspath(path.dirname(__file__))
+
+
+def main():
+
+    packages = ['svalinn_tools', 'img']
+    package_data = {'': ['*.png']}
+    setup_kwargs = {
+        "name": "dagmc-viz",
+        "version": VERSION,
+        "description": 'DAGMC Visualization Tools',
+        "long_description": open(path.join(here,'README.md')).read(),
+        "long_description_content_type": 'text/markdown',
+        "author": 'Svalinn Development Team',
+        "url": 'https://github.com/svalinn/DAGMC-viz',
+        "classifiers": ['Programming Language :: Python :: 2.7'],
+        "install_requires": ['numpy'],
```

Comment:
should this also require PyMOAB and VisIt?

Reply:
This is specifically for required Python packages, and it will try to install whatever is missing from PyPi, so I am not sure if I could put PyMOAB and VisIt here.